docs/src/lib/sets/SparsePolynomialZonotope.md

```diff
@@ -30,6 +30,7 @@ CurrentModule = LazySets.SparsePolynomialZonotopeModule
 ```@docs
 remove_redundant_generators(::SparsePolynomialZonotope)
 uniqueID(::Int)
+SparsePolynomialZonotopeModule.merge_id(::AbstractVector{Int}, ::AbstractVector{Int}, ::AbstractMatrix{N}, ::AbstractMatrix{N}) where {N}
```

Comment:
```suggestion
merge_id(::AbstractVector{Int}, ::AbstractVector{Int}, ::AbstractMatrix{N}, ::AbstractMatrix{N}) where {N<:Integer}
```
Actually, the module is already correct. I presume it does not match the `N` now.
If that does not work, you can also just try `merge_id`.